projects/ion/src/lib/chip/chip.component.html

```diff
@@ -0,0 +1,65 @@
+<button
+  data-testid="ion-chip"
+  [id]="chipId"
+  [disabled]="disabled()"
+  [class]="'ion-chip chip-' + size()"
+  [ngClass]="{
+    chip: !selected(),
+    'chip-selected': selected(),
+  }"
+  ionDropdown
+  [dropdownConfig]="{
+    shouldRender: !disabled() && hasDropdown(),
+    required: required(),
+    multiple: multiple(),
+    clearButton: multiple(),
+  }"
+  [dropdownOptions]="options() ? options() : []"
+  (dropdownOpened)="select()"
+  (dropdownEvent)="selectDropdownItem($event)">
+  @if (
+    (!hasDropdown() && icon()) ||
+    (hasDropdown() &&
+      dropdownWithIcon &&
+      iconPlaceholder &&
+      iconPosition() === 'left')
+  ) {
+    <ion-icon
+      [attr.data-testid]="'chip-icon-' + iconPosition()"
+      [attr.class]="'icon-color chip-icon-' + iconPosition()"
+      [type]="dropdownWithIcon ? iconPlaceholder : icon()"
+      [size]="size() === 'sm' ? 16 : 20"></ion-icon>
```

Comment:
```suggestion
      [size]="iconSize"></ion-icon>
```

Também chamaria uma função aqui para evitar esses dois valores sendo setados diretamente no HTML, dessa forma evitando até o problema de magic number.